src/py/mat3ra/made/tools/build/interface.py

```diff
@@ -10,29 +10,50 @@
 from ..convert import convert_atoms_or_structure_to_material, decorator_convert_material_args_kwargs_to_structure
 
 
-class SlabParameters(TypedDict):
-    MILLER_INDICES: Tuple[int, int, int]
-    THICKNESS: int
+DEFAULT_MAX_AREA = 400.0
 
 
-class ZSLParameters(TypedDict):
-    MAX_AREA_TOL: float
-    MAX_AREA: float
-    MAX_LENGTH_TOL: float
-    MAX_ANGLE_TOL: float
+class SlabParameters:
+    def __init__(self, miller_indices: Tuple[int, int, int] = (0, 0, 1), thickness: int = 3):
+        self.miller_indices = miller_indices
+        self.thickness = thickness
 
 
-class InterfaceParameters(TypedDict):
-    DISTANCE_Z: float
-    MAX_AREA: float
+class ZSLParameters:
+    def __init__(
+        self,
+        max_area: float = DEFAULT_MAX_AREA,
```

Comment:
The problem with using constant instead of a number literal is that documentation doesn't show the value:
![image](https://github.com/Exabyte-io/made/assets/79542055/1426bdfc-f29f-4002-a391-48928aab7e20)

Reply:
OK, use value